README.md

```diff
@@ -14,6 +14,8 @@ Dependencies
 Installation from sources:
 `setuptools` &ge; **0.6.14**
 
+In order to use `pg_activty` after an install using `pip`, you have to install the package `psycopg2` from `pgdg` first (see [here](https://github.com/dalibo/pg_activity/issues/118) for more info).
```

Comment:
I think we could reword it a bit like that:

>  you have to install the package `psycopg2`  from `pgdg` [APT](https://wiki.postgresql.org/wiki/Apt) or [YUM](https://yum.postgresql.org/) repositories. `psycopg2` can also been installed from `pip` with `pip install psycopg2` or `pip install psycopg2-binary` for the binary version.

Reply:
thx, works for me. fixed that way in [50ffa06](https://github.com/dalibo/pg_activity/pull/126/commits/50ffa06bd9d930fac85159ee79ac58154c759af9)